raw_data/preprocessed/africa/tanzania_english.txt

```diff
@@ -0,0 +1,769 @@
+
+Environmental protection plan and ecosystem management plan for Environmental Protected Areas 
+(48) [1] Where an area has been declared to be an Environmental Protected Area, the Council shall, in consultation with relevant sector prepare an environmental protection plan for that area. [2] The environmental protection plan shall: [a] set out the objectives of protecting and managing the area; [b] set out policies for the protection and management of the area; [c] formulate strategies for protection and management of the area; [d] provide for the development of social amenities and recreational facilities where necessary; [e] facilitate the carrying out of scientific research; and [f] incorporate all necessary requirements for environment management plans. [3] Notwithstanding provisions of the subsections [1] and [2], the Council may prepare an ecosystem management plan for an Environmental Protected Area. 
+
+[b] Environmental Management Plans for National Protected Areas 
+
+Environmental Management Plans for National Protected Areas
+(49) [1] Each national protected area shall be governed and managed in accordance with the respective written law under which it is protected. [2] Notwithstanding the provisions of subsection [1], each managing authority of a national protected area shall prepare and implement an environmental management plan which: [a] identifies areas of biological diversity; [b] identifies associated communities, other users and institutions to be involved; [c] describes extension and education work with communities and other users on the establishment of the protected areas; [d] indicates costs and benefits of the area's protection in a manner considered equitable by the person's identified under subsection [1][b]; [e] describes the boundaries of the national protected area; [f] defines the management measures to be taken within the national protected area; [g] outlines the regulations applicable to the national protected area; [h] prescribes means to monitor the implementation of the plan; [i] describes any other matter relevant to environmentally sound management of the national protected area; [j] is approved by the relevant authority having mandate and is under the jurisdiction of that protected area; and [k] is published, where appropriate, in the Gazette by the Minister responsible for the national protected areas systems. [3] Management measures that may be taken pursuant to subsection [1][f] may include: [a] zoning; [b] access restrictions; [c] use restrictions; [e] benefit sharing; [t] entrance fees and permits; and [g] any other measure deemed appropriate for proper and sound use of the area. 
```

Comment:
Please replace:
* `person's identified` with `persons identified`;
* `[e] benefit` with `[d] benefit`;
* `[t] entrance` with `[e] entrance`; and
* `[g] any other` with `[f] any other`.